lms/djangoapps/grades/tasks.py

```diff
@@ -54,8 +55,26 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
     abstract = True
 
 
+@task(base=_BaseTask)
+def compute_all_grades_for_course(**kwargs):
```

Comment:
I think we can factor out the body of the first for loop in `_shuffled_task_kwargs()` in the management command file into a helper method, then use that helper method here (as well as in the management command).